coreAPI/src/main/java/net/java/games/input/Event.java

```diff
@@ -38,39 +38,113 @@
  *****************************************************************************/
 package net.java.games.input;
 
+import java.util.Objects;
+
 public final class Event {
+	/** {@link Component} that generated this {@code Event}. */
     private Component component;
+
+	/**
+	 * <p>A value associated with this {@code Event}.</p>
+	 *
+	 * <p>
+	 *     This could represent any number of things, depending on the {@link Component} that generated this
+	 *     {@code Event}. Rotation of a joystick, pressure on a button, etc.
+	 * </p>
+	 */
     private float value;
+
+	/**
+	 * <p>This represents the time elapsed since the last {@code Event}, in nanoseconds.</p>
+	 *
+	 * <p>
+	 *     e.g. If an {@code Event} happens, then one millisecond later another {@code Event} happens, the {@code nanos}
+	 * 	   of the second {@code Event} will be 1,000,000 (1 millisecond).
+	 * </p>
+	 */
```

Comment:
This may be incorrect. I based the description off of what I believed the original documentation represented.